sklearn/cluster/_k_means.py

```diff
@@ -30,7 +30,7 @@
 from ..utils.validation import check_is_fitted, _check_sample_weight
 from ..utils.validation import FLOAT_DTYPES
 from ..exceptions import ConvergenceWarning
-from . import _k_means
+from . import _k_means_fast as _k_means
```

Comment:
I also find the naming `*_fast.pyx` files more meaningful than with underscores as it was done before.